Cubical/Algebra/CommAlgebra/UnivariatePolyList.agda

```diff
@@ -216,10 +219,30 @@ module _ (R : CommRing ℓ) where
             useSolver : (r : ⟨ R ⟩) → r ≡ (r · 1r) + 0r
             useSolver = solve R
 
+
     {- Reforumlation in terms of the R-AlgebraHom from R[X] to A -}
     indcuedHomEquivalence : AlgebraHom (CommAlgebra→Algebra ListPolyCommAlgebra) A ≃ ⟨ A ⟩
     fst indcuedHomEquivalence f = fst f X
     fst (fst (equiv-proof (snd indcuedHomEquivalence) x)) = inducedHom x
     snd (fst (equiv-proof (snd indcuedHomEquivalence) x)) = inducedMapGenerator x
     snd (equiv-proof (snd indcuedHomEquivalence) x) (g , gX≡x) =
       Σ≡Prop (λ _ → isSetAlgebra A _ _) (sym (inducedHomUnique x g gX≡x))
+
+
+  {- A corollary, which is useful for constructing isomorphisms to
+     algebras with the same universal property -}
+  isIdByUMP : (f : CommAlgebraHom ListPolyCommAlgebra ListPolyCommAlgebra)
```

Comment:
I think if we have `isIdByUMP` here as we do in `FreeCommAlgebra/Properties.agda`, then we should also factor it over an `equalByUMP` as done there. (But proved using `inducedHomUnique` in this case.)

Reply:
Done